services/ingestion/src/pdf_ingestion.py

```diff
@@ -146,8 +147,9 @@ def load_pdf_metadata(pdf_path: str, current_obj: Mapping[T, T]) -> Mapping[T, T
 
 @click.command()
 @click.argument('pdf_dir')
-def click_wrapper(pdf_dir: str) -> None:
-    run_pdf_ingestion(pdf_dir, insert_pdfs_mongo, insert_pages_mongo, run_ghostscript)
+@click.argument('num_processes')
```

Comment:
I'm not super familiar with click -- is this a required argument now? If so, we're missing it in the deployments.yaml, so the `./install.sh` setup would break.

Reply:
The deployment over kubernetes will need to be updated. For now I'm just writing commands that scan each other collections sequentially.